index.html

```diff
@@ -231,13 +224,6 @@
       manufacturer. They may also expose an <a>NDEF</a> message.
     </p>
   </div>
-  <p>
-    An <dfn>NFC peer</dfn> is an active, powered device which can interact
```

Comment:
Since it's an informative section, this info could stay with the note it's not supported any more because (link).

Reply:
Done in https://github.com/w3c/web-nfc/pull/524